crates/client/data-availability/Cargo.toml

```diff
@@ -42,7 +43,7 @@ mc-db = { workspace = true, default-features = true }
 starknet_api = { workspace = true, default-features = true }
 
 # Ethereum
-ethers = "2.0.7"
+ethers = "2"
```

Comment:
We're using a specific commit of ethers now in the workspace Cargo.toml. Should be fixed with a rebase.